e2e/cypress/integration/channel_sidebar/DM_category_spec.js

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @DM category
+
+import {getAdminAccount} from '../../support/env';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+const SpaceKeyCode = 32;
+const DownArrowKeyCode = 40;
+
+describe('MM-T3156 DM category', () => {
+    const sysadmin = getAdminAccount();
+    let testUser;
+    const usernames = [];
+    before(() => {
+        // # Enable channel sidebar organization
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                ExperimentalChannelSidebarOrganization: 'default_on',
+            },
+        });
+
+        // # Login as test user and visit town-square
+        cy.apiInitSetup({loginAfter: true}).then(({team, user}) => {
+            testUser = user;
+            cy.visit(`/${team.name}/channels/town-square`);
+
+            // # upgrade user to sys admin role
+            cy.externalRequest({user: sysadmin, method: 'put', path: `users/${user.id}/roles`, data: {roles: 'system_user system_admin'}});
+        });
+
+        // # Close "What's new" modal
+        cy.uiCloseWhatsNewModal();
+    });
+
+    it('MM-T3156_1 Should open DM modal on click of + in category header', () => {
+        cy.get('button[aria-label="DIRECT MESSAGES"]').parents('.SidebarChannelGroup').within(() => {
```

Comment:
This is fine but it's preferable to use the queries provided [testing-library](https://testing-library.com/docs/dom-testing-library/api-queries), so this can be:
```suggestion
        cy.findByLabelText('DIRECT MESSAGES').parents('.SidebarChannelGroup').within(() => {
```

Same for other similar case.